code/game/machinery/cryopod.dm

```diff
@@ -253,7 +253,12 @@
 		/obj/item/key,
 		/obj/item/door_remote,
 		/obj/item/autopsy_scanner,
-		/obj/item/holosign_creator/atmos
+		/obj/item/holosign_creator/atmos,
+		/obj/item/clothing/gloves/color/black/forensics,
+		/obj/item/rcd,
+		/obj/item/rpd,
+		/obj/item/gun/projectile/revolver/detective,
```

Comment:
Looking at line 240, `/obj/item/gun` is already protected. Not sure if this is necessary. However, what is might be good is putting shoulder holsters here instead. If you cryo with a gun in a shoulder holster I am *pretty* sure it gets lost.

Reply:
This was already noted to me by steelslayer, and was updated in the most recent commit (I think, I'm not entirely sure of what I'm doing yet)

Reply:
I should mention that even with holsters in this list, I'm not sure they will get found and stored if they're attached to a jumpsuit. I can't recall the cryopod code, and I can't look into this at the moment.